modular_skyrat/modules/customization/modules/food_and_drinks/food/scottish.dm

```diff
@@ -34,3 +34,28 @@
 	icon_state = "batteredsausage"
 
 
+/obj/item/food/cookie/shortbread
+	name = "shortbread"
+	desc = "A rectangular piece of cooked flour. Said to control the sun during Hogmanay."
+	icon = 'modular_skyrat/master_files/icons/obj/food/irnbru.dmi'
+	icon_state = "shortbread"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/sugar = 6)
+	tastes = list("sugar-y dough" = 1)
+	foodtypes = GRAIN | JUNKFOOD | SUGAR
+
+//HERE OUT OF LAZINESS/NOT SCOTTISH
```

Comment:
Put it in an appropriate file

Reply:
ok babe

Reply:
@tf-4 babe they're separate now...wanna kiss???